src/frontend/qt_sdl/LAN_Capture.cpp

```diff
@@ -0,0 +1,89 @@
+#include <stdio.h>
+#include <iostream>
+#include <sys/time.h>
+#include <filesystem>
+
+#ifdef __WIN32__
+	#include <ws2tcpip.h>
+#else
+	#include <sys/socket.h>
+	#include <netdb.h>
+	#include <poll.h>
+	#include <time.h>
+#endif
+
+#include "../types.h"
+#include "LAN_Capture.h"
+#include "Config.h"
+
+std::ofstream packetDumpOutput;
```

Comment:
please use C files instead.

Also global variables in PascalCase

Reply:
C files?
also, just fixed this style, among some other things you've mentioned. gonna commit those next

Reply:
https://en.cppreference.com/w/c/io

Reply:
you can read the new contributor guide: https://github.com/Arisotura/melonDS/blob/master/contributing.md